kani-driver/src/main.rs

```diff
@@ -41,6 +42,11 @@ fn cargokani_main(input_args: Vec<OsString>) -> Result<()> {
     args.validate();
     let ctx = session::KaniSession::new(args.common_opts)?;
 
+    if matches!(args.command, Some(CargoKaniSubcommand::Assess)) || ctx.args.assess {
```

Comment:
Is this the only alternative command we have?

Reply:
I think we will end up with more later, but it's the first.

For instance, I want us to have `cargo kani self-test` and `cargo kani versions`.

We do sort of have `cargo kani setup` but that's intercepted by the proxy binaries, not passed down to us.